hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerRatisRequest.java

```diff
@@ -58,6 +63,15 @@ public void testRequestWithNonExistentBucket()
 
     String volumeName = "vol1";
     String bucketName = "invalidBuck";
+
+    // Add entry to Bucket Table.
+    omMetadataManager.getBucketTable().addCacheEntry(
```

Comment:
Is this related to the patch?

Reply:
Yes, this test began failing due to a change in the internal implementation of getBucketLayout. So we need to add the volume to the volume table for this to work correctly.
I made a small error in this test change by adding a key to the bucket table instead, I have fixed that in my latest patch.